integration/airflow/tests/extractors/test_snowflake_extractor.py

```diff
@@ -264,3 +264,41 @@ def test_information_schema_query(get_connection):
         extractor._information_schema_query(different_databases_mixed)
         == different_databases_mixed_sql
     )
+
+@mock.patch("airflow.hooks.base.BaseHook.get_connection")
+@mock.patch('openlineage.airflow.extractors.snowflake_extractor.execute_query_on_hook')
+def test_extract_create_stage(execute_query_on_hook, get_connection):
+    Source(
+        scheme='snowflake',
+        authority='test_account',
+        connection_url=CONN_URI_URIPARSED
+    )
+
+    execute_query_on_hook.return_value = "public"
+
+    conn = Connection()
+    conn.parse_from_uri(uri=CONN_URI)
+    get_connection.return_value = conn
+
+    task = SnowflakeOperator(
+        task_id="snowflake_op_create_stage",
+        sql="""
+              CREATE STAGE IF NOT EXISTS IN_OPENLINEAGE.DATA
+              URL='gcs://openlineage/data/'
+              storage_integration = GCS_INT
+              FILE_FORMAT=(TYPE=PARQUET TRIM_SPACE=TRUE)
+              COPY_OPTIONS=(MATCH_BY_COLUMN_NAME=CASE_INSENSITIVE)
+        """,
+        parameters={"id": 57},
+    )
+
+    mock_get_hook(task)
+
+    get_hook_method(task).return_value._get_conn_params.return_value = {
+        'account': 'test_account',
+        'database': DB_NAME
+    }
+
+    metadata = SnowflakeExtractor(task).extract()
+    assert metadata.outputs[0].name == "FOOD_DELIVERY.IN_OPENLINEAGE.DATA"
+    assert metadata.inputs[0].name == "gcs://openlineage/data/"
```

Comment:
Are those two enough checks? Just asking because I'm not sure.

Reply:
well, I can add another one with `copy into` to test stage as output if you like